payloads/configure.go

```diff
@@ -102,3 +102,13 @@ type ConfigurePayload struct {
 type Configure struct {
 	Configure ConfigurePayload `yaml:"configure"`
 }
+
+// Init initialises instances of the Configure structure.
+func (conf *Configure) Init() {
```

Comment:
Could we call that one `InitDefaults()` and use it instead of `fillDefaults()` in the configuration package ?